src/components/Layout/types.d.ts

```diff
@@ -0,0 +1,4 @@
+export type LayoutProps = {
```

Comment:
Can you change this to interface? I think that it's better to use type only when you don't declare any props inside and you just want to extend some interfaces

Reply:
Done.